imap_processing/swe/l1b/swe_l1b_science.py

```diff
@@ -117,7 +118,7 @@ def deadtime_correction(counts: np.ndarray, acq_duration: int):
     return corrected_count
 
 
-def convert_counts_to_rate(data: np.ndarray, acq_duration: int):
+def convert_counts_to_rate(data: np.ndarray, acq_duration: float) -> np.ndarray:
```

Comment:
I wanted to leave this comment. We still want that as `int` because input should be integer.
```suggestion
def convert_counts_to_rate(data: np.ndarray, acq_duration: int) -> np.ndarray:
```

Reply:
@tech3371 
Let me know if the change I made will work. 
Mypy is flagging the type declared because the expression could technically return a float so it wants the variable to also match.

Reply:
Or I suppose another variable could be created that is a float and then set equal to the expression. That way `act_duration` is not being set equal to its manipulated self.

Reply:
I think it's ok. Can you add `TODO: check with SWE team` so that I don't forget to check this in upcoming discussion? Otherwise, it looks good to me!